docs/whats-new/toc.yml

```diff
@@ -3,8 +3,8 @@ items:
   href: index.yml
   expanded: true
   items:
-  - name: .NET 7 documentation updates
-    href: dotnet-7-docs.md
+  - name: .NET 8 documentation updates
+    href: ../core/whats-new/dotnet-8.md
```

Comment:
I don't think this is correct. This is supposed to point to doc updates, not product updates. So the whole thing should just be removed if we're not doing a docs updates page for .NET 8.

Reply:
Probably.